snyk-monitor/templates/deployment.yaml

```diff
@@ -188,6 +188,24 @@ spec:
           - name: SNYK_WORKERS_COUNT
             value: {{ quote .Values.workers.count }}
           {{- if .Values.sysdig.enabled }}
+          - name: SNYK_SYSDIG_RISK_SPOTLIGHT_TOKEN
+            valueFrom:
+              secretKeyRef:
+                name: snyk-sysdig-secret
+                key: token
+                optional: true
```

Comment:
Is there ever potential for another token? Should we call this one something more specific?

Reply:
I'd say it makes sense to be one token per secret, given that the secret hold info about 1 integration with the agent.